app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,63 @@
+class ArticlesController < ApplicationController
+  def index
+    articles=Article.all
+    render json: articles, status: 200
+  end
+
+  def show
+    article=Article.find(params[:id])
+    if article
+      render json: article, status: 200
+    else
+      render json:{
+        error: "error finding article"
+      }
+    end
+  end
+
+  def create
+    article=Article.new(
+      title: article_params[:title],
+      body: article_params[:body],
+      author: article_params[:author]
+    )
+    if article.save
+      render json: article, status: 200
+    else
+      render json: {
+        error: "Error Creating..."
```

Comment:
return custom errors instead of static error, 
return errors that are returned by model due to failure in save.

Reply:
implemented return of custom errors that are returned by model due to failure. please check the latest commit.